### PR TITLE
add sticky footer component

### LINKED ISSUE #9: Add: Sticky Footer
Footer text ideas: 
created with love by debugs bunnies

### PR DESCRIPTION
**Pull request by KimaruThagna**
Added as an independent component in the `footer/` folder.
The css handling the footer component has been added to the main index.css file
To use the component, the main index.js will import it and in the render function, use `<Footer />`
This solves issue #9

### CODE REVIEW

File: footer/index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import "../index.css"
+
+const Footer = () => {
+  return (
+    <div className="footer">
+      <div className="footerInner">
+        <p>
+          Copyright © 2021 <a href="https://gmcruz.me/debugs-bunnies/">DebugsBunnies POD 2.0.3 </a>  |{' '}
```
Comment (gmelodie) [inline footer/index.js]: Instead of this we should put "Made with :heart: by Debugs Bunnies pod 2.0.3"

Comment (KimaruThagna) [reply footer/index.js]: Updating

**Comment by APiligrim** [comment]
I'll have to do slight refactoring and then merge bc the current code base dosen't use React

**Review by APiligrim — APPROVED** [review]

**Comment by APiligrim** [comment]
I did slight refactoring because the current code base dosent use React, also changed the position of footer from fixed to relative just so that it dosent look as crowded on the page as the user scrolls and sees so many things on the page. We can change it back again if it looks better once the page is complete :-)

 Otherwise looks great!

**Pull request merged**